.github/workflows/check-pl-compat.yaml

```
@@ -111,7 +111,7 @@ jobs:
         CXX_COMPILER=$(which g++ }}) \
         ENABLE_LLD=OFF \
         RT_BUILD_DIR="$(pwd)/runtime-build" \
-        LIGHTNING_GIT_TAG_VALUE=master \
+        LIGHTNING_GIT_TAG_VALUE=0.35.0 \
```

Oh no not these sorry, they always stay the same
```suggestion
        LIGHTNING_GIT_TAG_VALUE=master \
```